src/content/en/agencies/partials/web-directory.html

```diff
@@ -0,0 +1,75 @@
+{% framebox class="directory-framebox" height="auto" enable_widgets="true" %}
+  <head>
+    <link rel="stylesheet" type="text/css" href="/web/agencies/css/directory.css">
+  </head>
+  <body>
+    <div class="filters">
+      <div class="filters__search">
+        <input class="filters__search__field" placeholder="Search for web agencies" type="text" name="agency-search" value="" autocomplete="off" aria-label="Search box">
+        <div class="devsite-search-image material-icons"></div>
+      </div>
+      <div class="filters__dropdowns"></div>
+    </div>
+    <div id="map"></div>
+    <div class="table__bar">
+      <div class="table__logo"></div>
+      <div class="table__info table__heading mobile-is-hidden" data-sort-key="agency_name" data-sort-by="alphabetically">
+        <span class="material-icons">arrow_right_alt</span>Agency name
+      </div>
+      <div class="table__lighthouse-score table__heading mobile-is-hidden" data-sort-key="max_lighthouse_score" data-sort-by="numerically" data-sort-ascending="false">
+        <span class="material-icons">arrow_right_alt</span>Highest Lighthouse Score
+      </div>
+    </div>
+    <div class="directory-framebox__wrapper"></div>
+    <script src="/web/agencies/js/directory.js" type="text/javascript"></script>
+    <script>
+      // Markup for each card below
```

Comment:
I assume you've tried this on real devsite and it works?

Reply:
Should have added this the description -- it does: https://devsite.googleplex.com/web/agencies/